docs/release-notes/2.39.0.md

```diff
@@ -0,0 +1,65 @@
+# Percona Monitoring and Management 2.38.0
+
+| **Release date:** | Aug 7, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+## Release Highlights
+
+### MongoDB backups: UI-based restore on a new replica set cluster
+
+Building on the recent MongoDB backup management enhancements, we have now added the option to restore backups on different clusters with the same configuration. This enables you to test restore data before deploying it on Dev environments. 
+Restoring on different clusters is not available for sharded clusters. This is a complicated process that still requires manual intervention. For more information, see [Restoring from a sharded cluster]()
+
+### Query Analytics - Meta information
+
+QAN is an important tool that can help filter data and identify problematic queries. However, it can be challenging to determine which services, nodes, and other labels are linked to these queries. PMM 2.39.0 now includes a metadata table for QAN to address this issue. This table contains information such as Node name, Node ID, Service name, Service Type, and user name for each selected query. With this feature, you can easily identify the services, nodes, and labels associated with your identified queries.
```

Comment:
```suggestion
QAN is an important tool that can help filter data and identify problematic queries. However, it can be challenging to determine which services, nodes, and other labels are linked to these queries. PMM 2.39.0 now includes a metadata table for QAN to address this issue. This table contains information such as Node name, Service name, Service Type, and user name for each selected query. With this feature, you can easily identify the services, nodes, and labels associated with your identified queries.
```

node ID is removed from the list